server/routes/gdc.topVariablyExpressedGenes.ts

```diff
@@ -111,3 +112,17 @@ async function getGenes(q: any, ds: any, genome: any) {
 		throw e
 	}
 }
+
+function tempGetCGCgenes(genome: any) {
+	const lst = [] as string[] // list of ENSG ids from cgc genes
+	// don't think there's need to preparse genome.geneset, as this function is only temporary
+	for (const s of genome.geneset[0].lst) {
+		const a = genome.genedb.getAliasByName.all(s)
+		if (a) {
+			for (const b of a) {
+				if (b.alias.startsWith('ENSG')) lst.push(b.alias)
```

Comment:
@xzhou82 Does this mean that a non-CGC genes would not have a `ENSG` alias? In other words, only CGC genes are expected to have `ENSG` aliases? Just asking for my own information.